test/server/config_validation/config_fuzz_test.cc

```diff
@@ -19,9 +19,17 @@ namespace {
 // mode (quits upon validation of the given config)
 DEFINE_PROTO_FUZZER(const envoy::config::bootstrap::v3::Bootstrap& input) {
   envoy::config::bootstrap::v3::Bootstrap sanitizedInput(input);
-  // TODO(asraa): QUIC is not enabled in production code yet, so remove references for HTTP3.
-  // Tracked at https://github.com/envoyproxy/envoy/issues/9513.
-  for (auto& cluster : *sanitizedInput.mutable_static_resources()->mutable_clusters()) {
+  for (::envoy::config::cluster::v3::Cluster& cluster :
```

Comment:
Not sure why this changed from `auto&`, we tend to use auto for protobuf due to verbosity and contextual clarity, but in any case, not worth blocking on.